README.md

```diff
@@ -1,2 +1,89 @@
-# mobile-menu-trial
-A mobile menu built with HTML, JS and CSS
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Portfolio Micro ](#-portfolio-micro-)
+  - [Live Link ](#live-link-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Portfolio Micro <a name="about-project"></a>
+
+**Portfolio Micro** A minimaly designed portfolio, created based on the structure of the portfolio templates collected by Microverse. This website was created taking as reference the template 1 from [this figma file](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
+
+## Live Link <a name="live-link"></a>
+
+https://github.com/wickathou/mobile-menu-trial/
+
```

Comment:
- [ ] I have just realized that your live demo link leads to the repository home page and not to a live demo of this project. KIndly update it with the updated version for it. You could use [GitHub pages](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site) for this.